### PR TITLE
APPEND node + RAND changes

### PR DESCRIPTION
**Pull request by dstrande**
### Description

- [x] I've included a concise description of what each node does

### Styleguide

- [x] My node adheres to the [styleguide](https://github.com/flojoy-io/nodes/blob/main/node_styleguide.md) for Flojoy nodes

### Docs

- [x] I've submitted a PR for a documentation page for the new node(s) that contains usage examples (see docs.flojoy.io)

### Testing

- [x] This PR includes a unit test ([example here](https://github.com/flojoy-io/nodes/blob/c28808f2d0aec8f116cdb3fccb46c4f5256574e9/TRANSFORMERS/ARITHMETIC/ADD/ADD_test_.py) and/or ideally a screenshot of the node's output on an example app.

### CODE REVIEW

**Comment by dstrande** [comment]
pytest of APPEND_test.py:

```
(flojoy202306) (base) dal@dal-desktop:~/Documents/flojoy/20230621/studio/PYTHON/nodes/LOGIC_GATES/LOOPS/APPEND$ pytest APPEND_test.py
============================================ test session starts =============================================
platform linux -- Python 3.10.9, pytest-7.1.3, pluggy-1.0.0
rootdir: /home/dal/Documents/flojoy/20230621/studio/PYTHON/nodes, configfile: pytest.ini
plugins: cov-4.1.0
collected 1 item                                                                                             
APPEND_test.py .                                                                                       [100%]
============================================= 1 passed in 0.46s==============================================
```

File: LOGIC_GATES/LOOPS/APPEND/APPEND.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    Returns
+    -------
+    Ordered pair (numpy arrays).
+    """
+
+    if len(dc_inputs) < 2:
+        raise ValueError(
+            f"To append, APPEND node requires two inputs, {len(dc_inputs)} was given!"
+        )
+    y0 = dc_inputs[0].y
```
Comment (scorinaldi) [inline LOGIC_GATES/LOOPS/APPEND/APPEND.py:24]: This assumes that the data containers will have `x` and `y` members, which I think means they're `ordered_pairs`? Could you make this node more generic, and have it potentially work with type matrix and or dataframe?

File: LOGIC_GATES/LOOPS/APPEND/APPEND_test.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    element_b = DataContainer(
+        type="ordered_pair", x=numpy.linspace(11, 12, 1), y=numpy.linspace(11, 12, 1)
+    )
+
+    # node under test
+    res = APPEND.APPEND([element_a, element_b], {})
+
+    # check that the correct number of elements
+    print(res.y, type(res.y), len(res.y))
```
Comment (scorinaldi) [inline LOGIC_GATES/LOOPS/APPEND/APPEND_test.py:42]: Can you remove this `debug` print statement?

**Review by scorinaldi — CHANGES_REQUESTED** [review]
Thanks for creating this node! I've left a note about a change that would make this node more useful.

File: LOGIC_GATES/LOOPS/APPEND/APPEND.py (hunk 1 above)
Comment (dstrande) [reply LOGIC_GATES/LOOPS/APPEND/APPEND.py:24]: I think this will probably be a different node (CONCAT) to work with pd.concat and np.concatenate.

**Comment by dstrande** [comment]
@scorinaldi I've made the changes requested and added tests for matrix and dataframe types.

```
(base) dal@dal-desktop:~/Documents/flojoy/20230627/studio/PYTHON/nodes/LOGIC_GATES/LOOPS/APPEND$ pytest APPEND_test.py
============================================ test session starts =============================================
platform linux -- Python 3.10.9, pytest-7.1.3, pluggy-1.0.0
rootdir: /home/dal/Documents/flojoy/20230627/studio/PYTHON/nodes, configfile: pytest.ini
plugins: cov-4.1.0
collected 1 item                                                                                             

APPEND_test.py .                                                                                       [100%]

============================================= 1 passed in 0.46s ==============================================
```

**Review by scorinaldi — APPROVED** [review]
LGTM thanks!

**Pull request merged**